packages/meteor-gazelle-createdby/lib/behavior/behavior.js

```diff
@@ -0,0 +1,109 @@
+// Based on the meteor-astronomy-timestamp-behavior package:
+// https://github.com/jagi/meteor-astronomy-timestamp-behavior/
+Astro.createBehavior({
+  name: 'createdby',
+  options: {
+    hasCreatedByField: true,
+    createdByFieldName: 'createdBy',
+    hasUpdatedByField: true,
+    updatedByFieldName: 'updatedBy',
+    hasCreatedAtField: true,
+    createdAtFieldName: 'createdAt',
```

Comment:
@daveyshindig  Wouldn't createdAt and updatedAt be part of the timestamp behavior?